### ISSUE TITLE
Einige Kurven erzeugen keine Schnittpunkte und werden daher nicht ausgestanzt

### ISSUE THREAD

**Issue by tordans** [gislars/strassenraum-berlin#4]
Es kommt vor, dass die Geometrie auf eine Art verläuft, die keinen Schnittpunkt erzeugen. An dieser Stelle kann dann die Park-Linie nicht gestanzt werden.

Beispiel:

![image](https://user-images.githubusercontent.com/111561/165975647-3ff61e2e-c273-482b-9155-5ff576387900.png)

Schnittpunkt https://www.openstreetmap.org/node/31259658

**Comment by joshinils** [comment]
da treffen 3 ways aufeinander

**Comment by SupaplexOSM** [comment]
Meinst du die "Innenkurve" rechts? Im Idealfall passiert das nicht, wenn Segmente mit gleichen Attributen zusammengefasst werden. Hier an der Stelle haben wir das Problem, dass sich die Linienrichtung in OSM ändert (die Straßen im Norden und im Südosten zeigen jeweils von der Kreuzung weg). Theoretisch könnte man aber auch das versuchen vor dem Versatz zusammenfassen.

Andererseits bin ich mir nicht sicher, ob man da nicht tatsächlich in der Kurve parken darf. Denn es gibt ja keine Einmündung. Quasi eine gebogene T-Kreuzung.

**Comment by tordans** [comment]
Da wir jetzt ja auch verlinken können, hier der Link zum Beispiel https://parkraum.osm-verkehrswende.org/project-vector-tiles/#18.73/52.4724289/13.4425294

**Comment by SupaplexOSM** [comment]
> Da wir jetzt ja auch verlinken können, hier der Link zum Beispiel https://parkraum.osm-verkehrswende.org/project-vector-tiles/#18.73/52.4724289/13.4425294

Das bringt mich auf die Idee #27.

(Meinst du denn jetzt die Kurve oder nicht? :D)

**Comment by tordans** [comment]
> (Meinst du denn jetzt die Kurve oder nicht? :D)

Ja, sorry: Diese zwei Linien, die normalerweise schneiden würden und damit eine Stanzung erzeugen würden/könnten.

<img width="401" alt="image" src="https://user-images.githubusercontent.com/111561/175787135-8167ebb8-f895-4ff3-9aa6-b246f1ad8349.png">

Mein Verständnis war, dass man laut StVo in Kurven nicht parken darf. Das würde IMO hier gelten. Aber evtl. ist das auch falsch… — Und selbst wenn es laut StVo so wäre, wird es in Berlin auf jeden Fall nicht durchgesetzt. Eine Beispiel ist auch https://www.openstreetmap.org/node/5396231503; die Kurve ist immer sehr unangenehm zugeparkt (von einem Fahrschulauto…)

**Comment by SupaplexOSM** [comment]
Kann sein, müssten wir uns nochmal nach der rechtlichen Lage erkundigen (die für uns maßgeblich sein sollte). Dafür bräuchte es dann aber bestimmt eine neue aufwendige technische Lösung z.B. mit Kurvenwinkeln/-distanzen o.ä., denn einen Bordsteinschnittpunkt gibt es da ja nicht.
